packages/signers/src/turnkey/__tests__/signer.test.ts

```diff
@@ -1,9 +1,17 @@
-import { WebauthnStamper } from "@turnkey/webauthn-stamper";
 import { TurnkeyClient } from "@turnkey/http";
-import { TurnkeySigner } from "../signer.js";
+import { WebauthnStamper } from "@turnkey/webauthn-stamper";
 import { custom } from "viem";
+import { TurnkeySigner } from "../signer.js";
 import { TurnkeySubOrganization } from "../types.js";
 
+vi.mock("@turnkey/viem", () => {
+  const createAccount = vi
+    .fn()
+    .mockResolvedValue("0x1234567890123456789012345678901234567890");
+
+  return { createAccount };
+});
```

Comment:
Actually, could we instead mock the `WebAuthnStamper` to return a specific value? that way we can still be sure to test the full flow